src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2464,16 +2464,16 @@ namespace NServiceBus.Sagas
     }
     public class SagaFinderDefinition
     {
-        public SagaFinderDefinition(System.Type type, string messageType, System.Collections.Generic.Dictionary<string, object> properties) { }
-        public string MessageType { get; }
+        public System.Type MessageType { get; }
+        public string MessageTypeName { get; }
```

Comment:
@timbussmann the class is intended to be used (and currently is in all test that dogfood) in combination with either SagaMetadataCollection or SagaMetdata.